dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/WorkflowExecuteRunnable.java

```diff
@@ -908,7 +908,7 @@ private boolean executeTask(TaskInstance taskInstance) {
             // todo: remove this method
             if (!processService.submitTask(workflowInstance, taskInstance)) {
                 log.error("Submit standby task: {} failed", taskInstance.getName());
-                return true;
+                return false;
```

Comment:
It seems that it should return false if it submits to DB failed, not sure if there are other considerations.
but there is also a todo here. cc @ruanwenjun 
will we remove this method or fix this logic

Reply:
It's better not to change this logic unless we find a bug related to this issue, when the workflow instance is READY_STOP, it will also submit the task, but will return false here to avoid dispatch the task.